cpp/community_detection_module/algorithm/louvain.cpp

```diff
@@ -104,7 +105,11 @@ void LoadUndirectedEdges(const mg_graph::GraphView<> &memgraph_graph, GrappoloGr
   for (std::size_t i = 0; i < number_of_vertices; i++) added[i] = 0;
 
     // Build the edgeList from edgeListTmp:
-#pragma omp parallel for
+#pragma omp parallel
+{
+  [[maybe_unused]] const enum mgp_error tracking_error = mgp_track_current_thread_allocations(graph);
```

Comment:
Do we need to track memory that isn't dynamically allocated? Maybe @antoniofilipovic could check what should we track in this algorithms?